app/src/main/java/com/android/partagix/model/ItemViewModel.kt

```diff
@@ -54,9 +55,25 @@ class ItemViewModel(
    * @param new the new item to update the UI state with
    */
   fun updateUiState(new: Item) {
+    var newUserId = ""
+    if (Authentication.getUser() != null) {
```

Comment:
Maybe there should be an else statement for a failsafe incase something goes wrong with the authentification ? Im not sure about wether this can happen or not.